x-pack/plugins/maps/public/classes/sources/es_geo_line_source/geo_line_form/size_slider.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useEffect, useState } from 'react';
+import useDebounce from 'react-use/lib/useDebounce';
+import { EuiRange } from '@elastic/eui';
+
+interface Props {
+  value: number;
+  onChange: (size: number) => void;
+}
+
+export function SizeSlider({ onChange, value }: Props) {
+  const [size, setSize] = useState(value);
+
+  const [, cancel] = useDebounce(
+    () => {
+      onChange(size);
+    },
+    150,
```

Comment:
Side note: We have so many different `wait` values for debouncing inputs in Maps. I don't think we need to change this. I just find it interesting.